vendor/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+import plistlib
+import shutil
+from xml.parsers.expat import ExpatError
+
+
+def get_file_content(filename):
+    with open(filename, 'r') as f:
+        return f.read()
+
+
+class HtmlBuilder:
+    """
+    Helper class to create html file from a report data.
+    """
+    def __init__(self, layout_dir):
+        self.layout_dir = layout_dir
+
+        self.header = get_file_content(
+            os.path.join(self.layout_dir, 'header.html'))
```

Comment:
There are a lot of duplications on joining these paths. Along with my "map" suggestion, I think this could be turned into something better. Instead of many variables, perhaps a dict, which is built with a helper function doing the joins.